### PR TITLE
Math magicians: full website

### PR DESCRIPTION
**Pull request by HENRYKC24**
### In this pull request, I did these:point_down:
1. I created a website consisting of 3 pages: Home, Calculator, and Quote
2. I added my own styles to improve the look and feel.
3. For every page, I created a route.
4. I made sure that navigation links for all 3 routes are displayed on each page.
5. I used React Router to make the routes work.
6. The general layout matchesd the wireframes.

### CODE REVIEW

File: src/components/Li.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  );
+};
+
+Li.propTypes = {
+  route: propTypes.string.isRequired,
+  name: propTypes.string.isRequired,
+  index: propTypes.number.isRequired,
+};
+
+export default Li;
```
Comment (Emmyn5600) [inline src/components/Li.js]: - [x] Kindly, rename the Li.js file using the meaningfully name.
rr7

File: src/pages/AppRoutes.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import Home from './Home';
+import Quotes from './Quotes';
+import PageNotFound from './PageNotFound';
+
+const AppRoutes = () => (
+  <>
+    <Navbar />
+    <Switch>
+      <Route exact path="/">
+        <Home />
```
Comment (Emmyn5600) [inline src/pages/AppRoutes.js]: - [x] The Home route or root route should be the last route, kindly implement it under `<pageNotFound>` route

File: src/pages/Home.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      sciences and technology, and in more recent times it has assumed a similar
+      role in the quantitative aspects of the life sciences.
+    </p>
+    <p className="para">
+      In many cultures—under the stimulus of the needs of practical pursuits,
+      such as commerce and agriculture—mathematics has developed far beyond
+      basic counting. This growth has been greatest in societies complex enough
+      to sustain these activities and to provide leisure for contemplation and
+      the opportunity to build on the achievements of earlier mathematicians.
+    </p>
```
Comment (Emmyn5600) [inline src/pages/Home.js]: - [x] Optional suggestion

- You can modify the font-size of the text to look nice as well as for us UI/UX purpose
rr6
.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @HENRYKC24 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
 - The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- The live link demo is working properly :heavy_check_mark:
- You have descriptive Readme file:heavy_check_mark:

Changes required :writing_hand: 

  - Please check the comment under the review.
  
  Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/components/Li.js (hunk 1 above)
Comment (HENRYKC24) [reply src/components/Li.js]: Done! Thanks:pray:

File: src/pages/AppRoutes.js (hunk 2 above)
Comment (HENRYKC24) [reply src/pages/AppRoutes.js]: I implemented it under the `pageNotFound` but the home page started giving "page not found". So I kept it just above `pageNotFound`. Thank you, @Emmyn5600 :pray:

File: src/pages/Home.js (hunk 3 above)
Comment (HENRYKC24) [reply src/pages/Home.js]: I appreciate. 
Thank you.

**Review by AshakaE — APPROVED** [review]
### Hi @HENRYKC24  🙋‍♀️, 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by HENRYKC24** [comment]
Thanks

**Pull request merged**